### PR TITLE
Adds GHA Workflow to actually Build the Agent across all the OS/Distro(s) we support today

### LINKED ISSUE #7919: The package uuid-dev (or libuuid-devel) has to be installed.
##### Question summary
I'm getting the following error when I run the installer:
```js
Sorry! netdata failed to build...

You may need to check these:

1. The package uuid-dev (or libuuid-devel) has to be installed.

   If your system cannot find libuuid, although it is installed
   run me with the option:  --libs-are-really-here

2. The package zlib1g-dev (or zlib-devel) has to be installed.

   If your system cannot find zlib, although it is installed
   run me with the option:  --libs-are-really-here

3. You need basic build tools installed, like:

   gcc make autoconf automake pkg-config

   Autoconf version 2.60 or higher is required.

If you still cannot get it to build, ask for help at github:

   https://github.com/netdata/netdata/issues
```
I tried to run the installer like this:
`./netdata/netdata-installer.sh --libs-are-really-here`
But it doesn't work.

When I run `apt-get install uuid-dev zlib1g-dev`, it shows the following:
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
uuid-dev is already the newest version (2.29.2-1+deb9u1).
zlib1g-dev is already the newest version (1:1.2.8.dfsg-5).
The following packages were automatically installed and are no longer required:
  net-tools python3-prettytable
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```
How can I fix this problem? I setup a lot of VPS with Netdata on Debian 9 and this never happened to me...
And of course, I read the other issues that seemed to be about the same, but I couldn't find anything that could help me...

##### OS / Environment
Debian 9.11

##### Expected results
Netdata should be installed.

### PR DESCRIPTION
**Pull request by prologic**
##### Summary

See #7919 where I found in repro that we indeed do not seem to properly
test that we can build the agent on all the different OS/Distro(s) we
support today. So this PR attempts to fix that which is similar to how
the Workflow in #7949 works.

##### Component Name

area/ci

##### Additional Information

Doesn't "fix" #7919 of course because our scripts that ensure correct
dev dependencies are installed seems to be broken for Debian 9 (stretch).

### CODE REVIEW

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4939184&bid=16330596)

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4939184&bid=16330596)

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4939184&bid=16330596)

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4939184&bid=16330596)

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          - 'debian:bullseye'
+          - 'debian:buster'
+          - 'debian:stretch'
+          - 'fedora:31'
+          - 'fedora:30'
+          - 'fedora:29'
+            #- 'opensuse/leap:15.2'
+            #- 'opensuse/leap:15.1'
+            #- 'opensuse/leap:15.0'
+            #- 'opensuse/tumbleweed:latest'
```
Comment (Ferroin) [inline .github/workflows/build.yml]: Is there some specific reason we're not handling OpenSUSE here?

Comment (prologic) [reply .github/workflows/build.yml]: It is broken for some reason. I will re-test later once merged to master and uncomment.

**Pull request merged**